Diagnostics/HealthChecker/Analyzer/Get-URLRewriteRule.ps1

```diff
@@ -0,0 +1,113 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
```

Comment:
We can, but I would like to be more consistent with when to have descriptions of functions inside the function vs outside the function. We should probably just define that somewhere and go from there.

I know what we need to have the description of the main script file to be prior to the `[CmdletBinding()]` section.

Do we have any good examples of during our build process where this is a bad course of action, other than when we accidentally put the description prior to the dependency files? 

@bill-long what is your take on this?

Reply:
I would like to see this formatted as comment help and put above the function:

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help

It's true that comment help _can_ go inside the function, but since I work in C# and PowerShell most of the time, and C# forces you to put it above the method, I like my PowerShell to look like C#.

Formatting it as comment help also allows you to call Get-Help on methods like this:

![image](https://github.com/microsoft/CSS-Exchange/assets/4518572/300af79c-8182-4bc0-a77b-545929a37eee)

Also, if we ever get to the point where we auto-generate docs for our scripts, the comment help for each method can be easily extracted.

Reply:
Okay, sounds legit. We should enforce this for all functions that have a description (if we have a chance to do so).

Reply:
Yeah, I wonder if we can create a rule or something like that for this. However, the trick is going to be the compliance header at the top of the file.

Reply:
Resolving this conversation for now. I'll file a work item to track this.

Reply:
Tracked in #1912